plugins/doc_fragments/purestorage.py

```diff
@@ -11,17 +11,17 @@
 class ModuleDocFragment(object):
```

Comment:
Also this doc fragment is not used within the collection. Does it also fall in the category "things that can be reused outside the collection"? Because if it doesn't, then we should kill it with fire. 

There is also a `module_utils/pure.py`, apparently related to this, also not used within c.g., but touching that one would require further investigation - not today.

Reply:
> Does it also fall in the category "things that can be reused outside the collection"?

Yes, it does.

It looks like we forgot to (deprecate and) delete the purestorage docs fragment and the pure module utils when we removed the associated modules (and/or other plugins, I don't remember right now). I guess we should deprecate them and delete them in the next major release. Marking them as deprecated with the meta/runtime.yml mechanism should be fine and sufficient here.

Reply:
Just submitted #9432